test/jdk/java/util/concurrent/tck/JSR166TestCase.java

```diff
@@ -431,7 +437,7 @@ protected void printFooter(TestResult result) {
      */
     static junit.textui.TestRunner newPithyTestRunner() {
         junit.textui.TestRunner runner = new junit.textui.TestRunner();
-        runner.setPrinter(new PithyResultPrinter(System.out));
+        //runner.setPrinter(new PithyResultPrinter(System.out));
```

Comment:
Please explain this change.

Reply:
I thought based on the `newPithyTestRunner ` comment there might be more information printed when running tests but I tried it by forcing one of the tests to fail locally and that doesn't really seem to be the case, I'll revert this.